website/docs/release_notes.md

```diff
@@ -7,6 +7,58 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v0.2.1](https://github.com/VirtusLab/scala-cli/releases/tag/v0.2.1)
+
+### Add a guide for migrating from old `scala` runner to Scala CLI
+
+As of [SIP-46](https://github.com/scala/improvement-proposals/pull/46), Scala CLI has been accepted as the new `scala`
+command. To make the transition smooth [@Gedochao](https://github.com/Gedochao)  prepared a [guide](./guides/old-runner-migration.md) highlighting
+the differences between the two runners.
```

Comment:
```suggestion
As of [SIP-46](https://github.com/scala/improvement-proposals/pull/46), Scala CLI has been accepted as the new `scala`
command. To make the transition smooth, we added a [guide](./guides/old-runner-migration.md) highlighting the differences between the two runners.
```
Make sure to use an absolute link to the guide on the actual release notes, as a relative link within the repo won't work there.